<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>%APPLICATION_NAME%</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/new/icon_resource/RS.ico</normaloff>
    <normalon>:/new/icon_resource/RS.ico</normalon>:/new/icon_resource/RS.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tab_widget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>521</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="connection_tab">
     <attribute name="title">
      <string>Connection</string>
     </attribute>
     <widget class="QGroupBox" name="instrument_info_group_box">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>80</y>
        <width>351</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>Instrument Info</string>
      </property>
      <widget class="QLineEdit" name="make_line_edit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="make_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>32</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Make:</string>
       </property>
      </widget>
      <widget class="QLabel" name="model_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>62</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Model:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="model_line_edit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>60</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="serial_no_line_edit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>90</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="serial_no_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>92</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Serial No:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="firmware_version_line_edit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>120</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="firmware_version_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>122</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Firmware Ver:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="frequency_range_line_edit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>150</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="frequency_range_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>152</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Freq Range:</string>
       </property>
      </widget>
      <widget class="QLabel" name="power_range_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>184</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Power Range:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="power_range_line_edit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>182</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="vna_connection_group_box">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>191</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>Connection Settings</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QPushButton" name="connect_push_button">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>90</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
      <widget class="QComboBox" name="connection_type_combo_box">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>GPIB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TCPIP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>USB</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="address_line_edit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="measure_tab">
     <attribute name="title">
      <string>Measure</string>
     </attribute>
     <widget class="QPushButton" name="measure_button">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>450</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Measure</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Results</string>
     </attribute>
     <widget class="QCustomPlot" name="custom_plot" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>741</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
